test/functional/lib/ipfs-request-workarounds.test.js

```diff
@@ -146,65 +148,23 @@ describe('modifyRequest processing', function () {
   })
 
   describe('a request to <apiURL>/api/v0/ with Origin=null', function () {
-    it('should remove Origin header ', async function () {
-      // set vendor-specific Origin for WebExtension context
+    it('should keep the "Origin: null" header ', async function () {
+      // Presence of Origin header is important as it protects API from XSS via sandboxed iframe
+      // NOTE: Chromium <72 was setting this header in requests sent by browser extension,
+      // but they fixed it since then.
```

Comment:
Removed unused code related to loading webui from gateway port without setting CORS and added test to ensure Companion does not remove `Origin: null` 

cc https://github.com/ipfs/go-ipfs-cmds/pull/190